pkg/deploy/dev-workspace-config/dev_workspace_config.go

```diff
@@ -148,6 +150,16 @@ func updateWorkspaceStorageConfig(devEnvironments *chev2.CheClusterDevEnvironmen
 	return nil
 }
 
+func updatePersistUserHomeConfig(persistentHomeConfig *chev2.PersistentHomeConfig, workspaceConfig *controllerv1alpha1.WorkspaceConfig) {
+	workspaceConfig.PersistUserHome = nil
+	if persistentHomeConfig != nil {
+		if workspaceConfig.PersistUserHome == nil {
+			workspaceConfig.PersistUserHome = &controllerv1alpha1.PersistentHomeConfig{}
+		}
+		workspaceConfig.PersistUserHome.Enabled = pointer.Bool(pointer.BoolDeref(persistentHomeConfig.Enabled, constants.DefaultPersistUserHomeEnabled))
```

Comment:
Can't we just set `workspaceConfig.PersistUserHome.Enabled = persistentHomeConfig.Enabled`?

Reply:
Correct :P For some reason I was hyperfixated on ensuring `persistentHomeConfig.Enabled` would default to `false` and forgot that setting it to `nil` (in the DWOC) would result in persistentHome being disabled.